hilti/toolchain/src/compiler/codegen/ctors.cc

```diff
@@ -119,9 +119,6 @@ struct Visitor : hilti::visitor::PreOrder<std::string, Visitor> {
     result_t operator()(const ctor::RegExp& n) {
         std::vector<std::string> flags;
 
-        if ( n.isAnchor() )
-            flags.emplace_back(".anchor = 1");
-
         if ( n.isNoSub() )
             flags.emplace_back(".no_sub = 1");
```

Comment:
.no_sub was revised from a :1 bitfield to a `bool` in the changes. Assigning it as `= 1` is unlikely to still be correct.
BTW, "`&no_sub` remains available to override the default choice" in the preamble is inaccurate, as the attribute is everywhere `&nosub`. (Maybe to reduce confusion, the struct member could be better similarly as .nosub).

Reply:
`= 1` still works, but I'll update (the change, btw, was due to bitfields not supporting initializing to default values, that's a C++20 feature).

Reply:
I leave the spelling as is as well, prefer it this way, even if not consistent between attribute and C++.